docker/Dockerfile

```diff
@@ -57,3 +61,6 @@ RUN git clone -b ${BRANCH} https://github.com/getyoti/yoti-drupal.git --single-b
         && cd ${DIRPATH}/sites/all/modules \
         && unzip ${PLUGIN_PACKAGE_NAME} \
         && rm -f ${PLUGIN_PACKAGE_NAME}
+
+EXPOSE 80
```

Comment:
Two ports?

Reply:
I was leaving http active as well which uses port 80. But I can disable that.

Reply:
Sure - I am only exposing port `443` for now.